activesupport/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+*   Deprecate `alias_method_chain` in favour of `Module#prepend` introduced in Ruby 2.0
```

Comment:
@kirs Rails tries to stick to american english, so next time prefer favor over favour etc. :smile: